src/main/webapp/general.css

```diff
@@ -101,3 +101,24 @@ footer .glyphicon {
     color: white;
     text-color: white;
 }
+
+/* Vertically align modal: http://jsfiddle.net/sRmLV/1140/ */
+.vertical-alignment-helper {
+    display:table;
+    height: 100%;
+    width: 100%;
+    pointer-events:none;
+}
```

Comment:
Does this make the page fill the whole screen?

Reply:
It helps to center the modal.